web-app/src/components/CreateUser/container.js

```diff
@@ -17,31 +25,140 @@ export default Wrapped =>
 
     initialFormState() {
       return {
-        forename: '',
-        surname: '',
-        email: '',
-        password: '',
-        country: 1,
-        status: employeeStatus.ACTIVE, //TODO Create enum file
-        employeeRole: 1,
-        startDate: moment(),
+        forename: {
+          label: 'Forename:',
+          elementType: 'input',
+          elementConfig: {
+            type: 'text',
+            placeholder: 'Enter your forename',
+          },
+          value: '',
+          validation: {
+            required: true,
+          },
+          valid: false,
+          touched: false,
+        },
+        surname: {
+          label: 'Surname:',
+          elementType: 'input',
+          elementConfig: {
+            type: 'text',
+            placeholder: 'Enter your surname',
+          },
+          value: '',
+          validation: {
+            required: true,
+          },
+          valid: false,
+          touched: false,
+        },
+        email: {
+          label: 'Email:',
+          elementType: 'input',
+          elementConfig: {
+            type: 'email',
+            placeholder: 'Enter your email address',
+          },
+          value: '',
+          validation: {
+            required: true,
+            isEmail: true,
+          },
+          valid: false,
+          touched: false,
+        },
+        password: {
+          label: 'Password:',
+          elementType: 'input',
+          elementConfig: {
+            type: 'password',
+            placeholder: 'Enter a password',
+          },
+          value: '',
+          validation: {
+            required: true,
+            minLength: 6,
+          },
+          valid: false,
+          touched: false,
+        },
+        country: {
+          label: 'Country:',
+          elementType: 'select',
+          elementConfig: {
+            options: [
+              { value: 1, displayValue: 'Northern Ireland' },
+              { value: 2, displayValue: 'Mexico' },
+            ],
+          },
+          value: 1,
+          validation: {},
+          valid: true,
+        },
+        status: {
+          //TODO Create enum file employeeStatus.ACTIVE
+          label: 'Status:',
+          elementType: 'select',
+          elementConfig: {
+            options: [
+              { value: employeeStatus.ACTIVE, displayValue: 'Active' },
+              { value: employeeStatus.INACTIVE, displayValue: 'Inactive' },
+            ],
+          },
+          value: 1,
+          validation: {},
+          valid: true,
+        },
+        employeeRole: {
+          label: 'Employee Role:',
+          elementType: 'select',
+          elementConfig: {
+            options: [
+              { value: 3, displayValue: 'Employee' },
```

Comment:
I assume this is something to do with defaulting to 'Employee' but can we put these in the right order?